test/e2e/workflow/deployment_test.go

```diff
@@ -454,6 +455,124 @@ var _ = Describe("NetworkAddonsConfig", func() {
 			})
 		})
 	})
+
+	Context("NetworkAddonsConfig labels", func() {
+		var (
+			configSpec            cnao.NetworkAddonsConfigSpec
+			ovsDaemonSet          = &v1.DaemonSet{}
+			kubemacpoolDeployment = &v1.Deployment{}
+		)
+
+		BeforeEach(func() {
+			configSpec = cnao.NetworkAddonsConfigSpec{
+				Ovs:         &cnao.Ovs{},
+				KubeMacPool: &cnao.KubeMacPool{},
+			}
+
+			CreateConfig(gvk, configSpec)
+			CheckConfigCondition(gvk, ConditionAvailable, ConditionTrue, 15*time.Minute, CheckDoNotRepeat)
+		})
+
+		getDaemonsetLabels := func(ds *v1.DaemonSet) map[string]string { return ds.GetLabels() }
+		getDaemonsetTemplateLabels := func(ds *v1.DaemonSet) map[string]string { return ds.Spec.Template.GetLabels() }
+		getDeploymentLabels := func(ds *v1.Deployment) map[string]string { return ds.GetLabels() }
+		getDeploymentTemplateLabels := func(ds *v1.Deployment) map[string]string { return ds.Spec.Template.GetLabels() }
```

Comment:
Please consider making those regular functions

Reply:
I think that simple getters can totally be in onelines, because they're very simple.
anything more complicated should as you say be in regular functions

Reply:
imo it is much cleaner and readable to not have inline functions
when you read the file, you dont need to see the functions as first thing,
you see the code, and later extend the view upon need